src/pages/mentorship/mentorship-courses-loader.ts

```diff
@@ -0,0 +1,12 @@
+import {
+  MentorshipCourseRouteKeys,
+  MentorshipDefaultRouteKeys,
+  mentorshipCourses,
+  mentorshipCoursesDefault,
+} from 'data';
+
+export const mentorshipCoursesLoader =
+  async (pageUrl: MentorshipDefaultRouteKeys | MentorshipCourseRouteKeys) => {
```

Comment:
Do we really need `async` here?

Reply:
No, I don't have to.
I looked at a [similar function](https://github.com/rolling-scopes/site/blob/main/src/entities/course/api/loader.ts) and assumed that in the future the download will be from a remote resource